feature-group-definitions/grid.yml

```diff
@@ -22,15 +33,19 @@ compat_features:
   - css.properties.grid-template
   - css.properties.grid-template-areas
   - css.properties.grid-template-columns
+  - css.properties.grid-template-columns.fit-content
+  - css.properties.grid-template-columns.minmax
   - css.properties.grid-template-columns.repeat
   - css.properties.grid-template-rows
+  - css.properties.grid-template-rows.fit-content
+  - css.properties.grid-template-rows.minmax
   - css.properties.grid-template-rows.repeat
-  - css.properties.justify-content
-  - css.properties.justify-items
-  - css.properties.justify-self
-  - css.properties.place-content
-  - css.properties.place-items
-  - css.properties.place-self
-  - css.properties.row-gap
+  - css.properties.justify-content.grid_context
+  - css.properties.justify-items.grid_context
+  - css.properties.justify-self.grid_context
+  - css.properties.place-content.grid_context
+  - css.properties.place-items.grid_context
+  - css.properties.place-self.grid_context
+  - css.properties.row-gap.grid_context
```

Comment:
The diff here fills in descendant features (e.g., adding `fit-content`) or targets the `grid_context` entries.

The features omitted from this list are:

- `subgrid`
- `masonry`
- grid animation features (see above)